### PR TITLE
Added mongoDB as BRVS verdict storage

### PR DESCRIPTION
**Pull request by Mingela**
There is a need to add integration test with mongoDB. For now you can change full pipeline integration test to use `MongoTransactionVerdictStorage` instead of `DummyMemoryTransactionVerdictStorage` but make sure that you deployed mongoDB instance at `localhost` (see `docker-compose.yml` for example of configuration)

### CODE REVIEW

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Observable<String> getRejectedTransactionsHashesStreaming() {
+    return subject;
+  }
+
+  private synchronized void store(String txHash, ValidationResult result) {
```
Comment (dolgopolovwork) [inline brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java]: Is it necessary to use `synchronized`?

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        replaceOptions
+    );
+  }
+
+  @Override
+  public void close() {
+    mongoClient.close();
+  }
+
+  public class MongoVerdict {
```
Comment (dolgopolovwork) [inline brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java]: I'd rather put it in another file

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java (hunk 1 above)
Comment (Mingela) [reply brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java]: Done

File: brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java (hunk 2 above)
Comment (Mingela) [reply brvs-core/src/main/java/iroha/validation/transactions/storage/impl/MongoTransactionVerdictStorage.java]: Done

File: deploy/docker-compose.yml [hunk 3]
```diff
@@ -2,15 +2,27 @@ version: '3.1'
 
 services:
   brvs:
-    container_name: brvs
+    container_name: "brvs"
```
Comment (x3medima17) [inline deploy/docker-compose.yml]: Why?

Comment (Mingela) [reply deploy/docker-compose.yml]: style consistency

**Pull request merged**